packages/cli/src/cli/commands/demo/print-gpt-prompt.ts

```diff
@@ -0,0 +1,40 @@
+import {demoStepsSchema} from '../../services/demo.js'
+import zodToJsonSchema from 'zod-to-json-schema'
+import Command from '@shopify/cli-kit/node/base-command'
+import {renderInfo} from '@shopify/cli-kit/node/ui'
+import {outputInfo} from '@shopify/cli-kit/node/output'
+
+export default class PrintGptPrompt extends Command {
```

Comment:
On one hand, I think it's probably a good idea to avoid using other companies' IP in our projects! OTOH, is `LLMPrompt` too obscure? What about `print-chat-ai-prompt`?

Reply:
I think what you landed on sounds good.